docs/setup/new_environment.md

```diff
@@ -327,6 +327,15 @@ If everything went well, you should now be able to access CommCareHQ from a brow
 
 If you are using virtualbox, see the [Configuring VirtualBox for testing CommCareHQ](../howto/configure-virtualbox.md) page to find the URL which depends on your networking setup.
 
+### Testing your new CommCare Environment
+
+Run the following command to test each of the backing services as described [here](../commcare-cloud/deploy.md#step-3-checking-services-once-deploy-is-complete).
+
+```
+cchq monolith django-manage check_services
```

Comment:
nit: every other command on this page is prepended by `$`.

Reply:
Thanks!
https://github.com/dimagi/commcare-cloud/pull/4671